aries-site/src/themes/aries.js

```diff
@@ -3,6 +3,40 @@ import { deepMerge } from 'grommet/utils';
 
 export const aries = deepMerge(hpe, {
   defaultMode: 'dark',
+  dataTable: {
+    pinned: {
+      header: {
+        extend: ({ theme }) => `
+        backdrop-filter: blur(8px);
+        @-moz-document url-prefix() {
+        background: ${
+          theme.dark ? 'rgba(64, 75, 92, 0.95)' : 'rgba(255, 255, 255, 0.95)'
```

Comment:
Can we translate these to the hexadecimal equivalents for consistency?

Reply:
How were these values arrived at? Did you have a chance to run these by @vavalos5 for a quick 👍 👎 ?

Reply:
I will change these and no I will run them by her I tried to use the `background-contrast` but it wasn't enough `opactity` still looked pretty bad

Reply:
@vavalos5 and I spoke and she is okay with these changes she just mentioned to make sure we stay on top of it when it is available for firefox so we can take this out!